Replace JARs with native binaries with ours. This isn't strictly
necessary, but it's better for security reasons and also to be sure that
the binaries will work correctly with musl.

netty-transport-native-epoll contains only native binary that is loaded
by transport-classes-epoll. It first attempts to load the binary from
'java.library.path' (/usr/lib, among others).

--- a/pom.xml
+++ b/pom.xml
@@ -950,7 +950,9 @@
       <dependency>
         <groupId>net.java.dev.jna</groupId>
         <artifactId>jna</artifactId>
-        <version>5.9.0</version>
+        <version>${alpine.jna.version}</version>
+        <scope>system</scope>
+        <systemPath>/usr/share/java/jna.jar</systemPath>
       </dependency>
       <dependency>
         <groupId>org.apache.lucene</groupId>
@@ -1329,7 +1329,9 @@
       <dependency>
         <groupId>com.github.luben</groupId>
         <artifactId>zstd-jni</artifactId>
-        <version>1.5.0-4</version>
+        <version>${alpine.zstd-jni.version}</version>
+        <scope>system</scope>
+        <systemPath>/usr/share/java/zstd-jni.jar</systemPath>
       </dependency>
       <dependency>
         <groupId>io.netty</groupId>
--- a/community/bolt/pom.xml
+++ b/community/bolt/pom.xml
@@ -88,17 +88,10 @@
 
         <dependency>
             <groupId>io.netty</groupId>
-            <artifactId>netty-transport-native-epoll</artifactId>
-            <classifier>linux-x86_64</classifier>
+            <artifactId>netty-transport-classes-epoll</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>io.netty</groupId>
-            <artifactId>netty-transport-native-epoll</artifactId>
-            <classifier>linux-aarch_64</classifier>
-        </dependency>
-
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
